src/main/java/net/snowflake/client/core/SFSession.java

```diff
@@ -1083,4 +1088,14 @@ private DriverPropertyInfo addNewDriverProperty(String name, String description)
   public boolean isAsyncSession() {
     return !activeAsyncQueries.isEmpty();
   }
+
+  @Override
```

Comment:
What is this overriding? I don't see this method defined in `SFBaseSession`.

Reply:
It is already exist. See line 798 in SFBaseSession.